elements/bulbs-poll/bulbs-poll.scss

```diff
@@ -39,11 +40,47 @@ bulbs-poll {
 
   }
 
+  .bulbs-poll-image-answers {
+    display: flex;
+  }
+
   .bulbs-poll-answers {
     margin: 0;
     padding: 0;
   }
 
+  li.bulbs-poll-image-answer {
```

Comment:
Let's not be specific as to the tag in these selectors. I've been following class or tag in this file, but not both.